deploy/olm/syncselector-template.yaml

```diff
@@ -22,7 +22,8 @@ objects:
     spec:
       clusterDeploymentSelector:
         matchLabels:
-          api.openshift.com/addon-reference-addon: 'true'
+          ext-hypershift.openshift.io/cluster-type: management-cluster
+          ext-hypershift.openshift.io/cluster-type: service-cluster
```

Comment:
We should use just:
```
api.openshift.com/managed: "true"
```
Nothing else, so it hits the entire fleet.

Reply:
https://issues.redhat.com/browse/SDA-6646?focusedCommentId=20924957&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-20924957

Reply:
After reading this comment, +1 to what @apahim is suggesting

Reply:
My understanding is the two labels listed here is a "AND" operation, there is no cluster will has these two labels together.

Reply:
we've concluded over slack that @apahim's proposal is the way to go.